core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -51,6 +51,26 @@
   protected long timestamp;
   protected boolean deleted;
 
+  /**
+   * Create a {@link Key} builder.
+   *
+   * @param copyBytes
+   *          if the bytes of the {@link Key} components should be copied
+   * @return the builder at the {@link KeyBuilder.RowStep}
+   */
+  public static KeyBuilder.RowStep builder(boolean copyBytes) {
```

Comment:
Well, this `KeyBuilder` is strict while the previous implementation was lazy which means that now I can't set the building option at the end. IMHO it's nice also in this way.